lisa/tools/xfstests.py

```diff
@@ -130,7 +130,8 @@ def _install_dep(self) -> None:
         # be installed if one of packages is not available in that distro,
         # so here install it one by one.
         for package in list(package_list):
-            posix_os.install_packages(package)
+            if posix_os.is_package_in_repo(package):
```

Comment:
What's the else? raise an exception?

Reply:
If not to raise an exception, add comments in code to explain it.

Reply:
added.